OutOfSchool/OutOfSchool.WebApi.Tests/Extensions/MapSearchResultToOkOrNoContentExtensionTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using OutOfSchool.BusinessLogic.Models;
+using OutOfSchool.BusinessLogic.Models.Application;
+using OutOfSchool.WebApi.Extensions;
+
+namespace OutOfSchool.WebApi.Tests.Extensions;
+
+[TestFixture]
+public class MapSearchResultToOkOrNoContentExtensionTests
+{
+    private class TestController : ControllerBase
+    {
+    }
+
+    private TestController controller;
+
+    [SetUp]
+    public void SetUp()
+    {
+        controller = new TestController();
+    }
+
+    [Test]
+    public void MapSearchResultToOkOrNoContentExtension_SearchResultIsNotNullAndNotEmpty_ReturnOk()
+    {
+        // Arrange
+        var applications = new List<ApplicationDto>() {
+            new ApplicationDto() { RejectionMessage = "Test 1" },
+            new ApplicationDto() { RejectionMessage = "Test 2", },
+            new ApplicationDto() { RejectionMessage = "Test 3" },
+        };
```

Comment:
(Test data generation) For test data you should use principle of YAGNI. 
I.e. here `RejectionMessage` value doesn't affect the test, thus you shouldn't fill it explicitely.
For efficient test data generation you could use Bogus library. I.e. to generate list of 3 applications you could use
`var applications = new Faker<ApplicationDto>().Generate(3);`

Please consider refactoring.

Reply:
Fixed with project generator.